src/ucp/tag/rndv.c

```diff
@@ -39,7 +39,7 @@ static void ucp_rndv_rma_request_send_buffer_dereg(ucp_request_t *sreq)
     if (UCP_DT_IS_CONTIG(sreq->send.datatype) &&
         ucp_ep_is_rndv_lane_present(sreq->send.ep, 0) &&
         ucp_request_is_send_buffer_reg(sreq)) {
-        ucp_request_send_buffer_dereg(sreq, ucp_ep_get_rndv_get_lane(sreq->send.ep, 0));
+        ucp_request_send_buffer_dereg(sreq);
```

Comment:
It seems that `sreq->send.lane` is AM. Then if AM and RNDV lanes are different, you will try to deregister memory on a wrong lane

Reply:
right, it was original code, after I found this I decided to remove this artefact argument

Reply:
Sorry it is deregister, not register